src/libraries/System.Private.CoreLib/src/System/Decimal.DecCalc.cs

```diff
@@ -153,13 +153,6 @@ private ulong Low64
                 1e80
             };
 
-            // Used to fill uninitialized stack variables with non-zero pattern in debug builds
```

Comment:
You mean instead of using 0xcc? Sure, I can change that. But the libraries seem a bit inconsistent. Some places uses 0xcc for this kind of debug filling and MSVC does it too, so that's why I went with it.